Mod0/syllabus.md

```diff
@@ -4,25 +4,33 @@ title:      Syllabus
 ---
 
 
-## Your Instructor
-The current Mod 0 Lead is Kaitlyn Van Gorkom, our Pre-Program Instructor at Turing. If you have any questions about the course or computer setup, Kaitlyn is the person to contact! You can find her on Slack with the handle **@kvg**. Kaitlyn and a team of Pre-Program Teaching Assistants run sessions and provide students feedback throughout Mod 0. They're an all-star team of Turing alumni and current software developers who have a passion for supporting students as they begin their journey at Turing.
+## Your Instructors
+The Launch Program instructors will guide you through the Mod 0 sessions:
+* Megan McMahon (Slack: @megan)
+* Richard Tillies (Slack: @ProfessorT)
+* Zoe Farrell (Slack: @zoe)
+If you have any questions about the course or computer setup, contact any of us via Slack.
+
 
 ## Description
 Mod 0 is a prerequisite course for attending Turing. The primary focus of the course is setting students up with the necessary tools and programming foundations to be successful in the full-time course. Students will be evaluated throughout the course on their communication, collaboration, and technical skills. 
 
 ## Computer Literacy Skill Requirements
 While computer literacy skills have never been an entry requirement for students at Turing, we recognize that some students may need to spend more time developing these skills to feel comfortable in this remote learning environment. Read through the following skills to identify where you might need to spend more time practicing before starting your Turing journey.
 
-- **Typing Speed:** It is recommended that students have an average typing speed of at least 50 words per minute before beginning their journey at Turing. You can test your current typing speed [here](https://www.typing.com/student/tests). If you find that you are not currently typing 50 words per minute or faster, this does not mean you cannot be successful at Turing, but it does mean you will need to improve this skill. Typing quickly and accurately without looking at the keyboard will make many things easier throughout your time at Turing.
+- **Typing Speed:** It is recommended that students have an average typing speed of at least 50 words per minute before beginning their journey at Turing. You can test your current typing speed [here](https://www.typing.com/student/tests). While not a requirement for the program, aim for a goal of typing 50 words per minute or faster. Typing quickly and accurately without looking at the keyboard will make many things easier throughout your time at Turing. Also, you will find that typing code is different than typing "regular" text (e.g. a letter or essay).
```

Comment:
Here is the 'code' typing site: https://typing.io/lesson/csharp/asp.net/CSharpCodeParser.cs/1